proposed/2021/MachineReadableOutput.md

```diff
@@ -0,0 +1,797 @@
+# Machine readable JSON output for dotnet list package
+
+* Status: **In Review**
+* Author: [Erick Yondon](https://github.com/erdembayar)
+* GitHub Issue [7752](https://github.com/NuGet/Home/issues/7752)
+
+## Problem background
```

Comment:
Great background! It really helps us understand the current limitations of the command today.